core/controllers/contributor_dashboard.py

```diff
@@ -316,8 +315,8 @@ def _get_reviewable_exploration_opportunity_summaries(
             for node in story.story_contents.get_ordered_nodes()
         ]
         in_review_suggestions, _ = (
-            suggestion_services
-            .get_reviewable_translation_suggestions_by_offset(
+            suggestion_services.
+            get_reviewable_translation_suggestions_by_offset(
```

Comment:
I think it makes sense to do the language filtering here, so that we only fetch the suggestions for the supplied language. Then, the below code can remain the same.

Reply:
Agreed with doing the language filtering here.

Reply:
I agree that it is a best approach to filter in the backend and only fetch the suggestions for the supplied language